macrame/src/test/scala/macrame/RefClassTest.scala

```diff
@@ -30,7 +30,7 @@ abstract class RefClassTest {
 }
 object RefClassTest {
    def apply[X[_] <: Ref[_], Y[_] <: Ref[_]](
-      id : Id[Post],
+      id :     Id[Post],
```

Comment:
That doesn't sit well with our code style. This is a forked repo, so, probably no reason it should, but still — are you sure aligning stuff like that makes sense?

Reply:
Scalariform upgrade did this. I'll look into whether I can do anything about it.

Reply:
Ouch. OK, good to know.

Reply:
Well actually this setting is enabled in this project:
```
.setPreference(AlignParameters, true)
```
So thing is that Scalariform did not correctly align parameters before, but now it does. So this formatting change actually sits well with this repo.